Bottom: bcf69a9d8a9001024f9d54c171ff7c304b659805
Top:    76cdd0e9552a923cbae25271c5115c5cca4a5306
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 16:40:53 +0000

Refresh of alternative-more-thorough-step

---

diff --git a/src/hid/common/object3d.h b/src/hid/common/object3d.h
index 4f40a3b..a7bae87 100644
--- a/src/hid/common/object3d.h
+++ b/src/hid/common/object3d.h
@@ -9,5 +9,4 @@ object3d *make_object3d (void);
 void destroy_object3d (object3d *object);
 object3d *object3d_create_test_cube (void);
 object3d *object3d_from_board_outline (void);
-void object3d_export_to_step (object3d *object, char *filename);
 void object3d_test_board_outline (void);
